scraper/scraper.js

```diff
@@ -12,6 +12,64 @@ const RARITY_BASE_URL = "https://animalcrossing.fandom.com/wiki/";
 const FOSSIL_URL =
   "https://animalcrossing.fandom.com/wiki/Fossils_(New_Horizons)";
 const ART_URL = "https://animalcrossing.fandom.com/wiki/Artwork_(New_Horizons)";
+const SEA_CREATURES_URL =
+  "https://animalcrossing.fandom.com/wiki/Deep-sea_creatures_(New_Horizons)";
+
+async function loadSeaCreatures() {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+  await page.goto(SEA_CREATURES_URL);
+
+  const result = await page.evaluate(() => {
+    const resultMap = {};
+    const bodies = Array.from(
+      document.querySelectorAll(".roundy.sortable tbody")
+    ).map((body) => Array.from(body.children, (r) => Array.from(r.children)));
+    for (row of bodies[0]) {
+      const name = row[0].textContent.trim();
+      const url = row[1].children[0].href;
+      const sellPrice = +row[2].textContent.trim().replace(",", "");
+      const size = row[3].textContent.trim();
+      const location = row[4].textContent.trim();
```

Comment:
Putting this in location is fine at the moment. I really can't think of a good icon to make for swimming pattern 🤷